index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SURVEY PAGE</title>
+</head>
+<body>
+    <link rel="stylesheet" href="style.css">
```

Comment:
- [ ] Good job so far 👏. Please **provide an apparent reason WHY you are suggesting this change**. You may consider providing links to some articles that explain that point very well 😉. 💡 Please edit the above comment which you have left before.